pkgs/development/python-modules/django-pydantic-field/default.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  django,
+  pydantic,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "django-pydantic-field";
+  version = "v0.3.12";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "surenkov";
+    repo = "django-pydantic-field";
+    rev = version;
+    hash = "sha256-rlnS67OGljWD8Sbyutb43txAH0jA2+8ju1ntSEP3whM=";
+  };
+
+  nativeBuildInputs = [ setuptools ];
+
+  propagatedBuildInputs = [
+    django
+    pydantic
+  ];
```

Enable the tests

```suggestion
  ];
  
	nativeCheckInputs = [
	  pytestCheckHook
	  pytest-django
	];
```

And you might need to set up `export DJANGO_SETTINGS_MODULE=tests.settings.django_test_settings` in `preCheck`.